MAUI/Scheduler/reminders.md

```diff
@@ -0,0 +1,423 @@
+        <syncfusion:SfScheduler.AppointmentMapping>
+            <syncfusion:SchedulerAppointmentMapping
+                    Subject="EventName"
+                    StartTime="From"
+                    EndTime="To"
+                    Background="Color"
+                    IsAllDay="IsAllDay"
+                    StartTimeZone="StartTimeZone"
+                    EndTimeZone="EndTimeZone"
+                    RecurrenceExceptionDates="RecurrenceExceptionDates"
+                    RecurrenceRule="RecurrenceRule"
+                    RecurrenceId="RecurrenceId"
+                    Reminders="Reminders">
+                <syncfusion:SchedulerAppointmentMapping.ReminderMapping>
+                    <syncfusion:SchedulerReminderMapping IsDismissed="IsDismissed"
+                    TimeBeforeStart="TimeBeforeStart"/>
+                </syncfusion:SchedulerAppointmentMapping.ReminderMapping>
+            </syncfusion:SchedulerAppointmentMapping>
+        </syncfusion:SfScheduler.AppointmentMapping>
+    </syncfusion:SfScheduler>
+{% endhighlight %}
+{% highlight c# tabtitle="ReminderViewModel.cs" %}
+public class ReminderViewModel 
+{
+  ...
+   public ObservableCollection<Event> Events { get; set; } = new ObservableCollection<Event>();
+
+   // Normal Appointment
+    Event normalAppointment = new Event()
+    {
+        From = DateTime.Now.AddMinutes(5),
+        To = DateTime.Now.AddHours(1),
+        EventName = "Normal Appointment",
+        Color = Brush.SkyBlue,
+        Reminders = new ObservableCollection<Reminder>
+        {
+            new Reminder {TimeBeforeStart = new TimeSpan (0,4,0)},
+        }
+
+    };
+    Events.Add(normalAppointment);
+
+    // All Day Appointment
+    Event allDayAppointment = new Event()
+    {
+        From = DateTime.Now.AddDays(1).AddMinutes(1),
+        To = DateTime.Now.AddDays(1).AddHours(1),
+        EventName = "All Day Appointment",
+        Color = Brush.SkyBlue,
+        IsAllDay = true,
+        Reminders = new ObservableCollection<Reminder>
+        {
+            new Reminder {TimeBeforeStart = new TimeSpan (0,0,50)},
+        }
+
+    };
+    Events.Add(allDayAppointment);
+
+    // Recurrence Appointment
+    Event recurrenceAppointment = new Event()
+    {
+        Id = 1,
+        From = DateTime.Now.AddDays(2).AddMinutes(1),
+        To = DateTime.Now.AddDays(2).AddHours(1),
+        EventName = "Recurrence Appointment",
+        Color = Brush.LightGray,
+        RecurrenceRule = "FREQ=DAILY;COUNT=3",
+        Reminders = new ObservableCollection<Reminder>
+        {
+            new Reminder {TimeBeforeStart = new TimeSpan (0,0,40)},
+        }
+
+    };
+    Events.Add(recurrenceAppointment);
+}
+{% endhighlight %}
+{% endtabs %}
+
+## ReminderAlertOpening event
+Scheduler notify the appointment's reminders by [ReminderAlertOpening] event before the appointment's start time. The [ReminderAlertOpeningEventArgs] has the following property,
```

Comment:
Scheduler notifies the appointment's reminders by [ReminderAlertOpening] event before the appointment's start time. The [ReminderAlertOpeningEventArgs] has the following property.